CalibMuon/DTCalibration/python/PlottingTools/plotResiduals.py

```diff
@@ -2,12 +2,22 @@
 from .fitResidual import fitResidual
 from .drawHistoAllChambers import drawHisto
 
-def plot(fileName,sl,dir='DQMData/Run 1/DT/Run summary/DTCalibValidation',option="HISTOPE1",draw=True):
+# This script is not to be run standalone.
+# Use CalibMuon/DTCalibration/scripts/allPlotsResiduals.py instead.
 
-    mean_ymin = -0.02
-    mean_ymax =  0.02
-    sig_ymin = 0.
-    sig_ymax = 0.07
+def plot(fileName,sl,dir='DTResiduals', run='1',option="HISTOPE1",draw=True):
+
+    if run in ['390106','390170']:
```

Comment:
Please confirm that this configuration is only run privately by the DT calibration experts, and it does not enter any central calibration workflow. If it is only privately run by DT experts it can even stay as such, with run numbers hardcoded in a nested configuration which lead to a lack of generalization, and certainly cannot allow this code to be run for any run number (because there are definitely other cosmics runs besides the few in the range of run numbers considered here).
Let me add also @battibass in case he has comments

Reply:
Yes, I confirm, this code is only run privately to produce some plots. There is no need to generalize it for the time being.